index.html

```diff
@@ -3610,7 +3610,7 @@ <h2>Definition of Roles</h2>
 					</tr>
 					<tr>
 						<th class="role-parent-head" scope="row">Superclass Role:</th>
-						<td class="role-parent"><rref>structure</rref></td>
+						<td class="role-parent"><rref>phrasingcontent</rref></td>
```

Comment:
I find this one confusing. The generic role specifically mentions `div` which does not match well.

Reply:
yet span does ;) - no idea for a solution though!